README.md

```diff
@@ -3,6 +3,6 @@
 This repo contains documentation for the .NET API Reference.
 
 The main repository for .NET documentation is the [.NET Docs repository](https://github.com/dotnet/docs). See the [Contributing Guide](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md) for information on updating .NET API reference documentation.
-We are tracking all work for this repository using [.NET Docs repository issues](https://github.com/dotnet/docs/issues). You can select the [Repo - dotnet-api-docs](https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22%3Afile_folder%3A+Repo+-+dotnet-api-docs%222) label to see issues that are scoped to this repository. You can also select the [Repo - samples](https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22%3Afile_folder%3A+Repo+-+samples%22) label to see issues that concern sample code.
+We are tracking all work for this repository using [GitHub isses](https://github.com/dotnet/dotnet-api-docs/issues). 
```

Comment:
Should we keep the part where we talk about samples? Some issues that will be filed here will concern sample code as well.